src/Assets/AssetsModule.php

```diff
@@ -560,6 +560,9 @@ protected function getPluginPath(string $path = ''): string
      */
     protected function enqueueIconSettings($current_section): void
     {
+        if (!strpos($current_section, 'mollie_wc_gateway_')) {
```

Comment:
Does it work? It will be `true` when it returns 0.

Usually it's better to check `=== false` for `strpos`.